editor/project_converter_3_to_4.cpp

```diff
@@ -2941,14 +2942,17 @@ void ProjectConverter3To4::rename_gdscript_functions(String &file_content) {
 				}
 			}
 		}
-		// -- start(a,b,c) -> start(a,Callable(b,c))      Thread
+		// -- start(a,b) -> start(Callable(a,b))      Thread
+		// -- start(a,b,c,d) -> start(Callable(a,b).bind(c,d))      Thread
```

Comment:
Threads in 3.x can take only one userdata argument. `start(a, b, c, d)` should become `start(Callable(a, b).bind(c), d)`, with `d` being the Thread priority.